src/backend/TrafficCourts/Workflow.Service/Services/OracleDataApiService.cs

```diff
@@ -31,17 +31,17 @@ public async Task<int> CreateDisputeAsync(Dispute disputeToSubmit)
 
             using (var httpClient = new HttpClient())
             {
-                var oracleDataApiBaseUri = $"http://{_oracleDataApiConfiguration.Host}:{_oracleDataApiConfiguration.Port}";
+                var oracleDataApiBaseUri = $"http://{_oracleDataApiConfiguration.Host}:{_oracleDataApiConfiguration.Port}/api/v1.0/";
```

Comment:
In a different PR we should convert this to a _oracleDataApiConfiguration.BaseUrl to include host, port and version path.